### PR TITLE
Fixed functions fetching company info / Security

### PR DESCRIPTION
**Pull request by GatoVuelta**
There was a little issue when running relative path installs which was about the default software & company branding, caused by the use of an absolute path by default.
This should be fixed now in this PR.
Small contribution 😃.

### CODE REVIEW

**Review by BelleNottelling — APPROVED** [review]

**Comment by andpavlenko** [comment]
Please setup your IDE to use PHP CS Fixer with config in project root dir.

File: src/bb-modules/System/Service.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ private function _getMultipleParams($params)
         $query = "SELECT param, value
                 FROM setting
                 WHERE param IN('".implode("', '", $params)."')
```
Comment (jaapmarcus) [inline src/bb-modules/System/Service.php:99]: Contains vulnrebility for a potential sql injection we need to filter those values before use in a query

**Review by jaapmarcus — COMMENTED** [review]
Not related to the PR it self but we should fix it anyway

**Comment by Komodroid** [comment]
Don't know why it is failing X_X can anyone check please? (Works fine on my end)

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**